src/howler.js

```diff
@@ -0,0 +1,7 @@
+var Howler
+
+if (typeof window !== 'undefined') {
+  Howler = require('howler')
+}
+
+module.exports = Howler
```

Comment:
No then you get always undefined, I think because the ES6 export will fire before the if statement.